src/main/java/org/generationcp/middleware/api/germplasmlist/GermplasmListGeneratorDTO.java

```diff
@@ -0,0 +1,144 @@
+package org.generationcp.middleware.api.germplasmlist;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import java.util.Date;
+import java.util.List;
+
+public class GermplasmListGeneratorDTO {
+
+	public static class GermplasmEntryDTO {
+
+		private Integer entryNo;
+		private Integer gid;
+		private String entryCode;
+		private String seedSource;
+		private String designation;
```

Comment:
@nahuel-soldevilla I think we should not allow sending the designation through the entries, I guess it should always be calculated based on the preferred name